Tasks/DockerComposeV0/task.json

```diff
@@ -14,7 +14,7 @@
     "version": {
         "Major": 0,
         "Minor": 150,
-        "Patch": 3
+        "Patch": 4
```

Comment:
in master this has to be one more than here, if m150 has 3, then your update will 4 and in master it should be 5. please do the same changes to other tasks as well.